bom/pom.xml

```diff
@@ -253,6 +253,31 @@
         <version>4.1.68.Final-SNAPSHOT</version>
```

Comment:
Could the Netty version be specified in a property so that you only have to modify it in one place?

Reply:
This one is updated automatically by the release plugin